platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -88,14 +88,16 @@ Enterprises adopting platform engineering can expect to achieve the following:
 
 1. Accelerate product development and delivery by intentionally dedicating teams
    to platform services and thereby reducing cognitive load on product teams
-1. Improve reliability and resiliency of products relying on platform
-   capabilities by dedicating experts to configure and manage them
-1. Accelerate product development and delivery by reusing and sharing platform
+2. Improve reliability and resiliency of products relying on platform
+   capabilities by dedicating experts to develop, configure and manage them
+3. Accelerate product development and delivery by reusing and sharing platform
    tools and knowledge across many teams in an enterprise
-1. Reduce risk of security, regulatory and functional issues in products and
+4. Easier and shorter onboarding of people moving between teams, as the platform 
```

Comment:
We'd considered this benefit to be part of the previous and mention it that way in the ensuing paragraph. I hear you though that it might be worth mentioning here. If so, let's make the language parallel to the other ones, something like: "Accelerate onboarding of current developers to new projects and teams by sharing the same platforms between them."

Can you suggest this in #305? Thanks!